api/structure/schema.py

```diff
@@ -0,0 +1,45 @@
+import graphene
+from graphene_sqlalchemy import SQLAlchemyObjectType
+
+from api.structure.models import Structure as StructureModel
+from helpers.auth.authentication import Auth
+from helpers.structure.create_structure import create_structure
+
+
+class Structure(SQLAlchemyObjectType):
+    """Autogenerated Return type for OfficeStructure"""
+
+    class Meta:
+        model = StructureModel
+
+
+class StructureInputs(graphene.InputObjectType):
+    web_id = graphene.String(required=True)
+    level = graphene.Int(required=True)
+    name = graphene.String(required=True)
+    parent_id = graphene.Int(required=True)
+    tag = graphene.String(required=True)
+    location_id = graphene.Int(required=True)
+    position = graphene.String(required=True)
+
+
+class CreateOfficeStructure(graphene.Mutation):
+    """ Returns payload on office structure creation"""
+    class Arguments:
+        data = graphene.List(
+            StructureInputs, required=True
```

Comment:
great work @daniellamarr, kindly add the description of `data` and also `structure`. An example is 
data = graphene.List(StructureInputs, required=True, description="Your description here"

Reply:
Okay I will